erigon-lib/types/transaction.go

```diff
@@ -152,7 +152,7 @@ func DecodeWrappedTransaction(data []byte) (Transaction, error) {
 	if data[0] < 0x80 { // the encoding is canonical, not RLP
 		return UnmarshalTransactionFromBinary(data, blobTxnsAreWrappedWithBlobs)
 	}
-	s := rlp.NewStream(bytes.NewReader(data), uint64(len(data)))
+	s := rlpStream(data)
```

Comment:
1. without return to pool 
2. what is difference with `rlp.DecodeBytes` method?

Reply:
- `done()` pattern allocating closure on each call 
- `rlp.DecodeBytes` also using `r := (*sliceReader)(&b)`

but anyway - current version is fine for me